index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 Boilerplate</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+	<h1>Hello Microverse!</h1>
+  </body>
```

Comment:
- [ ] As the previous reviewer asked you to add class to your `h1` tag as shown below :arrow_down:
`<h1 class="header">Hello Microverse</h1>`